measurements.py

```diff
@@ -220,9 +338,24 @@ class TimeMeasurer(AnalogMeasurer):
         measures=[positive_width, negative_width, positive_duty, negative_duty])
 
     def measure_range(self, data: AnalogSpan):
+        if data.clipped:
+            self.positive_width.error = CLIPPED_ERROR
+            self.negative_width.error = CLIPPED_ERROR
+            self.positive_duty.error = CLIPPED_ERROR
+            self.negative_duty.error = CLIPPED_ERROR
+            self.cycle_rms.error = CLIPPED_ERROR
```

Comment:
I'm not sure why width and duty cycle would get these errors but not frequency or period.
RMS makes sense for sure, but the error caused by clipping that affects width and duty cycle would affect frequency/period in the same way, notably that the midpoint voltage will be wrong.

What do you think?

Reply:
resolved, this is correct

Reply:
Discussed offline.